client/package.json

```diff
@@ -57,7 +58,8 @@
     ]
   },
   "devDependencies": {
-    "node-sass": "^4.13.1"
+    "node-sass": "^4.13.1",
```

Comment:
So we should remove node-sass and scss and Grommet and Bootstrap and only keep Ant and Styled Components, but we can do that in the next iteration and merge this first